kernel_tracing/metric_evaluation/no_dependencies.json

```diff
@@ -0,0 +1,12 @@
+{
+    "cases": [
+      {
+        "name": "Demo test",
+        "bpftrace": "./simple_bpftrace.bt",
+        "shell": ["sleep", "1"],
+        "conditions": [
+          "@val == 42"
+        ]
+      }
+    ]
+  }
```

Comment:
no idea what this does?

Reply:
It declares a test case for the python script to run. The help message in the python script explains how to execute it. I could add additional documentation if you think that makes sense.

Reply:
I don't remember the specific but for a very first example / demo I would generally err on the "too much documentation" side. Don't get me wrong: I mean "too much" as "rephrasing the same thing twice", NOT as "digressions about more advanced stuff".

Reply:
I added a readme in the metric_evaluation folder that explains what this all does.